modules/shared/src/main/scala/org/tessellation/currency/dataApplication/package.scala

```diff
@@ -313,6 +316,8 @@ object dataApplication {
   object DataApplicationBlock {
     implicit def decoder(implicit d: Decoder[DataUpdate]): Decoder[DataApplicationBlock] = deriveDecoder
     implicit def encoder(implicit e: Encoder[DataUpdate]): Encoder[DataApplicationBlock] = deriveEncoder
+
+    implicit val eq: Eq[DataUpdate] = (x: DataUpdate, y: DataUpdate) => x.equals(y)
```

Comment:
Is this correct? Something feels wrong about defining `Eq[DataUpdate]` here.
If this is a valid definition then shouldn't it be in the companion object for `DataUpdate` ?
Also, if using `.equals` is correct, should we be using this?
```
implicit val eq: Eq[DataUpdate] = Eq.fromUniversalEquals
```

Reply:
It is a leftover from some testing. Good catch. I removed that.